ls1intum/Artemis
# `General`: Fix local jenkins-gitlab setup

### Checklist
#### General
#### Server

### Motivation and Context
The local setup for jenkins and gitlab as specified in https://docs.artemis.ase.in.tum.de/dev/setup/jenkins-gitlab/ does not work currently and is quite complicated. 

### Description
This change simplifies the local setup (Gitlab/Jenkins) for developers.
It presets almost all values and should enable developers to get started in < 10 minutes.

### Steps for Testing
**Locally**: Follow the steps described in the setup guide and verify that the setup is working correctly.
This includes:
- Users & Passwords for Gitlab/Jenkins are working, as well as tokens are correctly generated
- Programming Exercises can be used (base build plans fail, solution build plan succeeds)

#### Code Review

#### Manual Tests
